src/gui/widgets/menu_button.cpp

```diff
@@ -241,6 +247,7 @@ namespace implementation
 
 builder_menu_button::builder_menu_button(const config& cfg)
 	: builder_styled_widget(cfg)
+	, update_label(cfg["update_label_"].to_bool(true))
```

Comment:
Config keys shouldn't have an underscore on the end.

Reply:
No, it shouldn't.